packages/mui-material/src/Select/Select.js

```diff
@@ -115,7 +115,7 @@ const Select = React.forwardRef(function Select(inProps, ref) {
         },
         ...(multiple && native && variant === 'outlined' ? { notched: true } : {}),
         ref: inputComponentRef,
-        className: clsx(InputComponent.props.className, className),
+        className: clsx(classes.select, InputComponent.props.className, className),
```

Comment:
In this component there is no `root` class. `select` is expected to be used instead.